source/_patterns/01-atoms/button/_button.twig

```diff
@@ -1,41 +1,44 @@
 {#
   Button!
```

Comment:
I haven't quite nailed it down yet, but something in this removes button styling from the Drupal side.